### DISCUSSION TITLE
More Reviews for PRO Users

### DISCUSSION THREAD

**Discussion by hunkardoner** [omkarcloud/google-maps-scraper#49] [General]
Is it possible more than 8 reviews for collected datas?

**Comment by Chetan11-dev** [comment]
May I know your use case for scraping reviews?

**Comment by AsmanyAkter** [comment]
I'm also a pro user too. I am scrapping data for the doctor's information. Is it possible to extract more than 8 reviews?

**Reply by Chetan11-dev** [reply]
Currently not possible

**Comment by Charlotte-br560** [comment]
Absolutely! To allow more than 8 reviews for collected data, consider checking out Crawlbase for potential solutions and insights. Their approach might provide useful guidance for enhancing the review functionality tailored for PRO users in your project.
